eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -43,16 +33,21 @@ steps:
   displayName: 'Apply Documentation Updates From Artifact'
   inputs:
     targetType: filePath
-    filePath: ${{ parameters.ScriptDirectory }}/release-docs-metadata.ps1
-    arguments: -ArtifactLocation ${{parameters.ArtifactLocation}} -Repository ${{parameters.PackageRepository}} -ReleaseSHA ${{parameters.ReleaseSha}} -RepoId ${{parameters.RepoId}} -WorkDirectory '${{parameters.WorkingDirectory}}' -DocRepoLocation "${{parameters.WorkingDirectory}}/repo" -Language "${{parameters.Language}}" -DocRepoContentLocation "docs-ref-services/"
+    filePath: ${{ parameters.ScriptDirectory }}/update-docs-metadata.ps1
+    arguments: -ArtifactLocation ${{parameters.ArtifactLocation}} -Repository ${{parameters.PackageRepository}} -ReleaseSHA ${{parameters.ReleaseSha}} -RepoId ${{parameters.RepoId}} -WorkDirectory '${{parameters.WorkingDirectory}}' -DocRepoLocation "${{parameters.WorkingDirectory}}/repo" -Language "${{parameters.Language}}" -DocRepoContentLocation ${{ parameters.DocRepoDestinationPath }}
```

Comment:
Could you turn this to multiple lines to help with approving the readability?

Reply:
Ah. Only reason I hadn't was because I thought the `arguments` needed to be a single line, not an array. Thought this was a yml

Reply:
yml supports 2 types of line splitting folded (`>`) and literal (`|`) see https://yaml-multiline.info/. So I think you can change this to something like:

```
arguments: >
  -ArtifactLocation ${{parameters.ArtifactLocation}}
  -Repository ${{parameters.PackageRepository}}
  -ReleaseSHA ${{parameters.ReleaseSha}}
  -RepoId ${{parameters.RepoId}}
  -WorkDirectory '${{parameters.WorkingDirectory}}'
  -DocRepoLocation "${{parameters.WorkingDirectory}}/repo"
  -Language "${{parameters.Language}}"
  -DocRepoContentLocation ${{ parameters.DocRepoDestinationPath }}
```